packages/ibm-products/src/components/SteppedAnimatedMedia/SteppedAnimatedMedia.js

```diff
@@ -0,0 +1,153 @@
+    const localRef = ref ?? backupRef;
+    // load animation source
+    useEffect(() => {
+      const isJsonFile = (filePath) => filePath.includes('.json');
+      async function loadArtifact() {
+        if (filePaths && filePaths.every(isJsonFile)) {
+          const allJsonArtifacts = filePaths.map((el) => fetch(el));
+          const rawResponse = await Promise.all(allJsonArtifacts);
+          const jsonResponse = rawResponse.map((el) => el.json());
+          const fileData = await Promise.all(jsonResponse);
+          setJsonData(fileData);
+        }
+      }
+      loadArtifact();
+    }, [filePaths]);
+
+    useEffect(() => {
+      const prefersReducedMotion = window?.matchMedia
+        ? window.matchMedia('(prefers-reduced-motion: reduce)').matches
+        : true;
+      if (localRef.current) {
+        animRef.current?.destroy();
+        animRef.current = lottie.loadAnimation({
+          container: localRef.current,
+          renderer: 'svg',
+          animationData: jsonData[clamp(playStep, 0, jsonData.length - 1)],
+          loop: false,
+          autoplay: false,
+          rendererSettings: {
+            preserveAspectRatio: 'xMidYMid slice',
+          },
+        });
+        prefersReducedMotion
+          ? animRef.current?.goToAndStop(0)
+          : animRef.current?.goToAndPlay(0);
+      }
+
+      return () => animRef.current?.destroy();
+    }, [jsonData, localRef, playStep]);
+
+    if (!jsonData) {
+      return null;
+    }
+
+    return (
+      <div
+        {
+          // Pass through any other property values as HTML attributes.
+          ...rest
+        }
+        className={cx(
+          blockClass, // Apply the block class to the main HTML element
+          className, // Apply any supplied class names to the main HTML element.
+          // example: `${blockClass}__template-string-class-${kind}-n-${size}`,
+          {
+            // switched classes dependant on props or state
+            // example: [`${blockClass}__here-if-small`]: size === 'sm',
+          }
+        )}
+        ref={localRef}
+        role="main"
+        {...getDevtoolsProps(componentName)}
+      />
+    );
+  }
+);
+
+// Return a placeholder if not released and not enabled by feature flag
+SteppedAnimatedMedia = pkg.checkComponentEnabled(
+  SteppedAnimatedMedia,
+  componentName
+);
+
+// The display name of the component, used by React. Note that displayName
+// is used in preference to relying on function.name.
+SteppedAnimatedMedia.displayName = componentName;
+
+// The types and DocGen commentary for the component props,
+// in alphabetical order (for consistency).
+// See https://www.npmjs.com/package/prop-types#usage.
+SteppedAnimatedMedia.propTypes = {
+  /**
+   * Optional class name for this component.
+   */
+  className: PropTypes.string,
+  /**
+   * The file path(s) to json formatted Lottie animation files.
+   */
+  filePaths: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
In `EmptyStates`, I believe we use `illustration`, which doesn’t really convey that it’s a path/URL ... `UserProfileImage` uses `image`. We usually try to keep consistent, but since this is internal we can probably change it later without a breaking change. 🤔